### DISCUSSION TITLE
How to style the day of the week for the first month

### DISCUSSION THREAD

**Discussion by rmsnk** [gpbl/react-day-picker#1865] [Support]
### Bug description

Hi, awesome library, but I have found a one small bug, I want to highlight current day of the week, I did it using the following code 

```js
  const formatters: Partial<Formatters> = useMemo(() => {
    return {
      formatWeekdayName: (date) => {
        return (
          <span
            className={isToday(date) ? "active" : ''}
          >
            {format(date, "EEEEE")}
          </span>
        );
      },
    };
  }, []);
  ```
  
  but in the second month day is also highlighted.

### To reproduce

[Link to reproduce](https://codesandbox.io/s/react-daypicker-forked-lvn33h?file=/src/App.tsx)

### Expected behavior

Day is highlighted only in 1 month

### Screenshots

<img width="642" alt="Screenshot 2023-08-03 at 10 24 34" src="https://github.com/gpbl/react-day-picker/assets/113971806/c6851f2f-077a-4090-aceb-4112cad4dbed">

**Comment by gpbl** [comment]
Here the behavior on the formatter is correct, as it is applied to all the day headers.

You could pass a `HeadRow` but this would require to rewrite [HeadRow](https://github.com/gpbl/react-day-picker/blob/18cc13c84145442c463677aa943d4eb64c344f68/src/components/HeadRow/HeadRow.tsx) as [custom components](https://react-day-picker.js.org/guides/custom-components)

```tsx
import { DayPicker, HeadRow } from 'react-day-picker';

function CustomHeadRow(props: HeadRowProps) {
  return <HeadRow ...props>
}

<DayPicker customComponents={{ HeadRow: CustomHeadRow }} />
```

You could also target that 6th cell in the 1st table with a CSS like this - have you considered this option?

```css
.rdp-month:nth-child(1).rdp-head_cell:nth-child(6) {
  color: red;
}
```
